distribution/pom.xml

```diff
@@ -198,12 +199,6 @@
         </profile>
         <profile>
             <id>release-nacos</id>
-            <dependencies>
```

Comment:
why remove this dependecies?

Reply:
Repeated reference. The current `pom.xml` has referenced it globally.

Reply:
ok